src/Infopanel/Forstørrelse.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
burde kjøre `svgo` for å komprimere svg-filene

Reply:
Ok, done